bin/check-workflow-run.js

```diff
@@ -67,11 +67,11 @@ async function checkWorkflowRun(repoOwner, branch) {
     }
 
     if (!results.ci) {
-      console.log('Latest ci workflow run result was not \'completed\' and \'success\'.')
+      console.log("Latest ci workflow run result was not 'completed' and 'success'.")
     }
 
     if (!results.smokeTest) {
-      console.log('Latest smoke-test workflow run result was not \'completed\' and \'success\'.')
+      console.log("Latest smoke-test workflow run result was not 'completed' and 'success'.")
```

Comment:
This is a silly change. :P

Reply:
ah yea prettier favors changing the quotes to avoid escaping